### PR TITLE
Import (relevant parts of) .github from aep-dev/aep.dev.

### PR DESCRIPTION
**Pull request by rofrankel**
Also, modify the build-and-publish-protos workflow to reflect different directory structure in this repo.

The presence of a `buf.work.yaml` complicates the changes a bit; I didn't just remove `common-components/` but also `proto/`.  I don't know if this is correct overall, but it does make `buf build` pass (whereas `buf build proto` fails).

I also don't know why we need proto/aep-conformance in this repo.

And finally, I used a personal token for `BUF_TOKEN` because I can't figure out if Buf supports org tokens.

@Alfus would appreciate input on all of the above.

### CODE REVIEW

File: .github/CODEOWNERS
Comment (toumorokoshi) [inline .github/CODEOWNERS:2]: FYI it looks like this group isn't recognized. indentation issue?

**Review by toumorokoshi — APPROVED** [review]
proactively approved! assuming codeowners will be fixed.

File: .github/CODEOWNERS
Comment (rofrankel) [reply .github/CODEOWNERS:2]: It was a repo settings issue; the team didn't have access.  Fixed.

**Pull request merged**
